### ISSUE TITLE
Plugins with Editor Buttons not compatible

### ISSUE THREAD

**Issue by Marcao8** [turnermm/ckgedit#432]
**Is it possible to add buttons from other plugins to the ckgeditor as well?** They are only visible in the standard editor for me. 
See: https://i.imgur.com/zGqZpky.jpg
Compared to the DW Editor:
https://i.imgur.com/iKlotlo.jpg

Example plugins used, are:
https://www.dokuwiki.org/plugin:button
https://www.dokuwiki.org/plugin:datepicker
https://www.dokuwiki.org/plugin:do
https://www.dokuwiki.org/plugin:watchcycle

All hints on how to add them are greatly appriciated.

**Comment by turnermm** [comment]
No.  Ckedditor and dokuwiki editor are independent of each other.  You have to write special plugins for the ckeditor, like the tag, wrap, and geshi plugins.
